### PR TITLE
add IIIF on Mastodon

### PR DESCRIPTION
**Pull request by julsraemy**
A list to find people and projects active in the IIIF Community on Mastodon

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ If you have a link or a resource to add, you have a few options:
 Likewise, there are other ways to see how people are implementing IIIF.
 - [Github Topics page](https://github.com/topics/iiif?o=desc&s=updated) - This page will show you the Github repositories that self-identify as IIIF-related, with the most recently updated projects listed first. 
 - [IIIF Community on Zenodo](https://zenodo.org/communities/iiif/) - A Zenodo community for gathering articles and data related to IIIF.
+- [People, projects and institutions active within the IIIF Community on Mastodon](https://julsraemy.ch/mastodiiif/)
```
Comment (triplingual) [inline readme.md:22]: It would be good to get something in there about self-identification, as the GitHub Topics entry has.

Comment (julsraemy) [reply readme.md]: Thanks. I've just edited my commit.

**Comment by julsraemy** [comment]
ping @triplingual, does it look okay now?

File: readme.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ If you have a link or a resource to add, you have a few options:
 Likewise, there are other ways to see how people are implementing IIIF.
 - [Github Topics page](https://github.com/topics/iiif?o=desc&s=updated) - This page will show you the Github repositories that self-identify as IIIF-related, with the most recently updated projects listed first. 
 - [IIIF Community on Zenodo](https://zenodo.org/communities/iiif/) - A Zenodo community for gathering articles and data related to IIIF.
-- [People, projects and institutions active within the IIIF Community on Mastodon](https://julsraemy.ch/mastodiiif/)
+- [IIIF on Mastodon](https://julsraemy.ch/mastodiiif/) - People, projects and institutions that self-identify as being active within the IIIF Community on  Mastodon, the social network that is part of the Fediverse.
```
Comment (triplingual) [inline readme.md:22]: Ah -- I left a comment on a different commit, I think. (I do not comment lines often. My apologies.)
----
I like the first part, but the added ending raises more questions than it answers, at least to me. I'd prefer just ending at "Mastodon". Or maybe, if you want to explain Mastodon briefly, "Mastodon, a federated social network."

**Comment by julsraemy** [comment]
Thanks @triplingual. I've modified the ending.

**Comment by triplingual** [comment]
Added 👍🏻 to this PR

**Pull request merged**